app/views/pages/moving-guide/tdy.html.erb

```diff
@@ -1,11 +1,132 @@
-<%- @page_title = 'What to expect… TDY' -%>
+<%- @page_title = 'What to Expect with a TDY Move' -%>
 
 <%= render partial: 'shared/contextual_header', locals: {page_title: @page_title, parent_pages: []} %>
 
 <div class="usa-section main-section">
-  <h1>What to expect… <%= abbr_tag('tdy') %></h1>
+  <h1>What to expect with a <%= abbr_tag('tdy') %> Move</h1>
 
-  <%# TODO Add page content %>
+  <div class="usa-width-one-fourth">
+    <aside class="sidenav">
+      <div class="sidenav-jump-to">
+        Jump to:
+      </div>
+      <ul class="usa-sidenav-list">
+        <li><%= link_to 'Shipping TDY Household Goods', '#shipping' %></li>
+        <li><%= link_to 'Weight Allowances for TDY Shipments', '#weight_allowances' %></li>
+        <li><%= link_to 'TDY Shipments', '#shipments' %></li>
+      </ul>
+    </aside>
+  </div>
 
-  <p>Coming soon!</p>
+  <div class="usa-width-three-fourths">
+    <div id="shipping">
+      <h4>Shipping TDY Household Goods</h4>
+      <p>If you find yourself heading out on Temporary Duty for an extended period of time (31 or more days), your Approving Official may authorize you to ship a small amount of household goods, to include some of your professional gear and medical equipment. Your TDY shipment should be scheduled on the Defense Personal Property System (DPS) and as with any PCS visit your local Personal Property Office if you have any questions.</p>
+    </div>
+    <div id="weight_allowances">
+      <h4>Weight Allowances for TDY Shipments</h4>
+      <p>The member’s grade on the day travel begins determines which weight allowance is allowed.  The weight allowances below are in addition to any unaccompanied or accompanied baggage.  Furthermore, in the event the TDY location becomes the permanent duty station, this would mean that the TDY weight allowance would also be an additional allowance to any PCS weight entitlements.</p>
+
+      <p>Check out the table below. You’ll find out just how much you are allowed to ship to this location.</p>
+
+      <table>
+        <thead>
+          <tr>
+            <th scope="col">Military Ranks</th>
+            <th scope="col">Public Health Services</th>
+            <th scope="col">Weight Allowance (lbs)</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>O-10</td>
```

Comment:
Could the first cell in each row be a `<th scope="row">`?

Reply:
i thought about that but decided against it because the first 2 columns are equally important. also it automatically made the text bold which we didn't want.

Reply:
There's semantic/accessibility value in marking up table rows with headers, so if it makes sense from a data point of view, it'd be worth using `<th>`s here and adjusting the styling in CSS (or rolling with the default USWDS styling and not worrying about it 😂 ). The choice is up to you based on your familiarity with the content. Definitely not a show-stopper.

Reply:
does it make sense to have 2 columns with th in a single row?

Reply:
Ah, okay, gotcha. I didn't fully understand the nature of the content. Probably best to leave the markup as you have it (using `<td>`s).